app/src/main/java/com/vero/tasky/ui/theme/Type.kt

```diff
@@ -30,6 +30,16 @@ val Typography = Typography(
         fontWeight = FontWeight.Normal,
         fontSize = 16.sp
     ),
+    body2 = TextStyle(
+        fontFamily = Inter,
```

Comment:
The typography object, has a parameter called "DefaultFontFamily" (or smth like that)
Where you can set the fontfamily once, and gets applied everywhere
So you dont need to manually pass it to each body

I think Philipp doesnt have a video about it, but I have this one (in spanish). Maybe it can give you an idea: https://youtu.be/mIG_KEGr7Pc